target/linux/ramips/dts/mt7628an_xiaomi_mi-router-4a-100m-intl-v2.dts

```diff
@@ -0,0 +1,93 @@
+// SPDX-License-Identifier: GPL-2.0-or-later OR MIT
+
+#include "mt7628an_xiaomi_mi-router-4.dtsi"
+#include <dt-bindings/leds/common.h>
+
+/ {
+	compatible = "xiaomi,mi-router-4a-100m-intl-v2", "mediatek,mt7628an-soc";
+	model = "Xiaomi Mi Router 4A (100M International Edition V2)";
+
+	aliases {
+		led-boot = &led_power_yellow;
+		led-failsafe = &led_power_yellow;
+		led-running = &led_power_blue;
+		led-upgrade = &led_power_yellow;
+		label-mac-device = &ethernet;
+	};
+
+	leds {
+		compatible = "gpio-leds";
+
+		led_power_blue: power_blue {
+			color = <LED_COLOR_ID_BLUE>;
+			function = LED_FUNCTION_POWER;
+			gpios = <&gpio 11 GPIO_ACTIVE_LOW>;
+		};
+
+		led_power_yellow: power_yellow {
+			color = <LED_COLOR_ID_YELLOW>;
+			function = LED_FUNCTION_POWER;
+			gpios = <&gpio 44 GPIO_ACTIVE_LOW>;
+		};
+
+		wan {
+			color = <LED_COLOR_ID_BLUE>;
+			function = LED_FUNCTION_WAN;
+			gpios = <&gpio 37 GPIO_ACTIVE_LOW>;
+		};
+	};
+
+};
+
+&partitions {
+	partition@60000 {
+		label = "overlay";
+		reg = <0x60000 0x100000>;
+		read-only;
+	};
+
+	partition@160000 {
+		label = "firmware";
+		reg = <0x160000 0xea0000>;
+		compatible = "denx,uimage";
+	};
+};
+
+&pcie {
+	status = "okay";
+};
+
+&eeprom_factory_8000 {
```

Comment:
Where is the definition which is overwritten here? 

I think you should extend the `factory` partition definition like it is done here:
target/linux/ramips/dts/mt7628an_xiaomi_mi-router-4a-100m-intl.dts

Reply:
It can be found at (target/linux/ramips/dts/mt7628an_xiaomi_mi-router-4.dtsi)[https://github.com/openwrt/openwrt/blob/c609c3bd6ef4b7ba7ddcd346c87e2cff3602067f/target/linux/ramips/dts/mt7628an_xiaomi_mi-router-4.dtsi#L63], included at line 3.

Reply:
Sorry I was looking at the 23.05 branch.